src/activity_handlers.cpp

```diff
@@ -3805,6 +3812,16 @@ void activity_handlers::study_spell_finish( player_activity *act, player *p )
     }
 }
 
+void activity_handlers::scan_book_finish( player_activity *act, player *p )
+{
+    item &book = *act->targets[0];
+    item &mc = *act->targets[1];
+    mc.item_tags.insert( "MC_HAS_DATA" );
+    mc.set_var( "MC_BOOK", book.typeId() );
+    p->add_msg_if_player( _( "You meticulously scan %s and store it on sd card." ), book.tname() );
```

Comment:
I suggest call it everywhere either `SD-Memory card` (like item called), or `memory card` (to be more generic).